### PR TITLE
Generate separate .pc files for kj and kj-async

### PR DESCRIPTION
**Pull request by szenker**
This would allow applications to use kj only as the promise implementation is really interesting also for projects which don't want to make RPC calls.

### CODE REVIEW

File: c++/kj-async.pc.in [hunk 1]
```diff
@@ -0,0 +1,10 @@
+prefix=@prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: Cap'n Proto
```
Comment (kentonv) [inline c++/kj-async.pc.in]: Probably the name should be "KJ Async Framework Library", and for kj.pc "KJ Framework Library".

**Comment by kentonv** [comment]
This seems like a good idea! Just a nitpick regarding the titles.

**Comment by szenker** [comment]
Ok, I improved the name field in the .pc files.

**Comment by kentonv** [comment]
Thanks!

File: c++/capnp.pc.in [hunk 2]
```diff
@@ -1,11 +1,11 @@
 prefix=@prefix@
-exec_prefix=@exec_prefix@
```
Comment (kentonv) [inline c++/capnp.pc.in]: I missed this when reviewing this before, but removing these lines breaks these files. #289 fixes it.

Comment (szenker) [inline c++/capnp.pc.in]: Why does it break the build? Exec_prefix is for architecture dependent files.

Comment (szenker) [inline c++/capnp.pc.in]: Ok, my question was stupid: 
The directory for object files and libraries of object code. Do not install executables here, they probably ought to go in $(libexecdir) instead. The value of libdir should normally be /usr/local/lib, but write it as $(exec_prefix)/lib. (If you are using Autoconf, write it as ‘@libdir@’.)

**Pull request merged**